xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererMediaCodecSurface.cpp

```diff
@@ -212,13 +217,14 @@ void CRendererMediaCodecSurface::ReorderDrawPoints()
     dstRect.x1, dstRect.y1, dstRect.Width(), dstRect.Height(),
     adjRect.x1, adjRect.y1, adjRect.Width(), adjRect.Height());
 
+  // Assumption: SurfaceFlinger starts PTS processing on arrival of first frame.
+  // To have the best match between kodi / android clock we start on first RenderUpdate call.
+  // Beside this the first frame should not be rendered before the SurfaceView is resized.
```

Comment:
No. 
FlipPage is called from CRenderManager::FrameMove which is GUI
RenderUpdate is called from CRenderManager::PresentSingle which is Video.

The exact same logic is used in RenderAML and MMALRenderer.
I don't know why you want it different here.

User-ex is not the point here, or you'd just have left the glClear